src/application-delegate.coffee

```diff
@@ -66,13 +66,33 @@ class ApplicationDelegate
     ipc.send("call-window-method", "setFullScreen", fullScreen)
 
   openWindowDevTools: ->
-    remote.getCurrentWindow().openDevTools()
+    new Promise (resolve) ->
+      process.nextTick ->
```

Comment:
Basically because any synchronous interaction with the DevTools makes the issue reappear (i.e. we cannot even call `isDevToolsOpened`). I am not sure about what exactly means to interact with the DevTools, but there must be something odd happening in that code path either in Electron or in Chromium.